### DISCUSSION TITLE
I'm getting circular import issue

### DISCUSSION THREAD

**Discussion by Gamma-Software** [python-telegram-bot/python-telegram-bot#3828] [Q&A] [answered]
Hey, 
I started using the lib and look cool but I already faced the issue after importing the lib like so:
```python
from telegram import Update
from telegram.ext import ApplicationBuilder, ContextTypes, CommandHandler
```

I got a circular import issue
```bash
Traceback (most recent call last):
  File "/mnt/data/workspace/ai_projects/playbooks/telegram.py", line 1, in <module>
    from telegram import Update
  File "/mnt/data/workspace/ai_projects/playbooks/telegram.py", line 1, in <module>
    from telegram import Update
ImportError: cannot import name 'Update' from partially initialized module 'telegram' (most likely due to a circular import) (/mnt/data/workspace/ai_projects/playbooks/telegram.py)
```
I'm using python 3.10 and whatever the lib version, it appears.

**Comment by lemontree210** [comment]
Hey! Please look at [this section](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#why-am-i-getting-importerror-cannot-import-name-xy-from-telegram) in our Wiki.

**Reply by Gamma-Software** [reply]
Hum I don't have telegram installed 
![Capture d’écran 2023-08-05 à 14 24 46](https://github.com/python-telegram-bot/python-telegram-bot/assets/6960910/02fa4664-a897-411b-aeaa-806ab2d7dad2)

**Reply by Gamma-Software** [reply]
But I do have named the file telegram.py. That's it ! Sorry for the dumb discussion. I'll reguraly check the FAQ

**Reply by lemontree210** [reply]
I see, could you post the contents of the module you're trying to run? If it's short, you can do it here, otherwise you can use pastebin. Make sure you don't post your Bot token along with the code.

**Reply by Gamma-Software** [reply, marked as answer]
No need, the issue was I named the python file `telegram.py`
